lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -29,13 +29,15 @@
                      * @returns {HtmlSnippet} - HTML list for course-wide discussion topics.
                      */
                     getCourseWideDiscussionsHtml: function(courseWideDiscussions) {
-                        var subCategoryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
+                        var entryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
```

Comment:
As discussed, we shouldn't use both `entryTemplate` (entry) and `#cohort-discussions-subcategory-tpl` (subcategory).  I think you should discuss with @muzaffaryousaf to better understand the naming of the underscore templates.  Maybe there is a special reason that entries are being treated as subcategories in this case.  

First clarify understand.  Then possibly update names.  Maybe entryTemplate is better as subCategoryTemplate even if it contains entries?  I don't know at this point.

Once all is clarified, add clarifying comments where appropriate.  Possible both in the underscore template(?) as well as here.